Procfile

```
@@ -1,4 +1,5 @@
 web: bundle exec puma -p 3000 -C ./config/puma.rb
-job: bundle exec sidekiq -q default -q critical -q tasker
+critical_job: bundle exec sidekiq -q critical,2 -q tasker,1
```

After reading up about the polling interval, I think one job would be fine if we did something like this:
```suggestion
job: bundle exec sidekiq -q critical,4 -q tasker,3 -q default, 2 -q low,1
```

Thoughts?

Also, I wonder how the Procfile and our ASG is set up. Do we need to reconfigure it to look for the two jobs rather than just "job"?

---

Either would be an improvement on what we are currently doing! Introducing more processes may not be necessary, we can try out your suggested change and adjust as needed.